stdlib/math.jou

```diff
@@ -68,14 +72,25 @@ declare asinh(x: double) -> double
 declare atanh(x: double) -> double
 
 # Exponential and logarithmic functions
-# returns e^x
+# Returns e^x
 declare exp(x: double) -> double
-# returns 2^x
+declare expf(x: float) -> float
+
+# Returns 2^x
 declare exp2(x: double) -> double
+declare exp2f(x: float) -> float
+
+# Returns 10^x
+declare exp10(x: double) -> double
+declare exp10f(x: float) -> float
+
 # Returns the natural logarithm of x.
```

Comment:
While we're here, we might as well clean up this comment. It only applies to `log()` and `logf()`. Other functions should have similar comments with the correct logarithm base, e.g. `# Returns the base 2 logarithm of x.` for `log2()` and `log2f()`.